src/test/java/org/junitpioneer/testkit/assertion/suite/TestSuiteFailureAssert.java

```diff
@@ -24,7 +24,12 @@
 	 * @param exceptionTypes the exception types you want to check
 	 * @return a {@link ListAssert} for asserting exception messages
 	 */
-	ListAssert<String> withExceptionInstancesOf(Class<? extends Throwable>... exceptionTypes);
+	/*
+	 * Note: Parameter type should be `Class<? extends Throwable>...`,
+	 * but @SafeVarargs is currently not permitted on abstract methods,
+	 * which causes annoying warnings on every usage of this method
+	 */
+	ListAssert<String> withExceptionInstancesOf(Class<?>... exceptionTypes);
```

Comment:
Please let me know if this should be reverted. The alternative would be to use `@SuppressWarnings("unchecked")` for every caller of this method.

Reply:
I understand the impetus, but being able to pass arbitrary classes is weird. I think there's a better solution: All callers of this method only pass one argument, so let's refactor to:

```java
	/*
	 * Note: We avoid a varargs-variant of this method to prevent heap-pollution warnings.
	 * If you need a method with n exception types, create n-1 `withExceptionInstancesOf`
	 * (note the plural) overloads with [2, n] parameters.
	 */
	ListAssert<String> withExceptionInstanceOf(Class<? extends Throwable> exceptionType);
```

(Note the method name's singular.)

Reply:
Have kept the plural form name <code>withExceptionInstance<ins>s</ins>Of</code> because the method can still return multiple exceptions in case they all have the same type.